PopupBridgeExample/build.gradle

```diff
@@ -7,14 +7,18 @@ android {
     buildToolsVersion '29.0.3'
 
     defaultConfig {
-        applicationId 'com.braintreepayments.popupbridge.example'
+        applicationId 'com.braintreepayments.popupbridge.demo'
```

Comment:
This doesn't have to happen right now, but maybe it would make sense to rename this directory to `PopupBridgeDemo` instead of `PopupBridgeExample`?

Reply:
Yeah that makes sense. We're looking to standardize the "Demo" application name across each repo going forward.

Reply:
We decided to just go for it. The app module is now named Demo.